doc/TAN-5687-webhooks/CHANGES.md

```diff
@@ -0,0 +1,153 @@
+# Changes Made to Implementation Plan
```

Comment:
Was it intended that everything in this folder was committed? Maybe it was part of the working process and we should leave it out? Or is this information valuable enough to remain in the codebase?